dev/playbooks/workstation.yml

```diff
@@ -0,0 +1,11 @@
+
+- hosts:
```

Comment:
I think the problem is that not all roles should run as root
some installations and file operations should run with currente logged user.